app/src/features/rules/modals/MV3MigrationModal/MigrationModal/modal.scss

```diff
@@ -0,0 +1,363 @@
+    line-height: 150%; /* 22.5px */
+    margin-bottom: 1.5rem;
+  }
+
+  .description .text {
+    font-size: 14px;
+    color: #ccc;
+  }
+
+  /* Migrated Rules Styles */
+
+  .migrated-rules.section {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: var(--space-6, 16px);
+    align-self: stretch;
+
+    .header {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+      gap: 4px;
+      align-self: stretch;
+
+      .migrated-rules.title {
+        display: flex;
+        align-items: center;
+        gap: 8px;
+        align-self: stretch;
+        color: var(--requestly-color-warning);
+
+        .icon {
+          text-align: center;
+          font-family: "Material Symbols Outlined";
+          font-size: 16px;
+          font-style: normal;
+          font-weight: 400;
+          line-height: 16px; /* 100% */
+        }
+
+        font-style: normal;
+        font-weight: 700;
+        line-height: 150%;
+      }
+    }
+
+    .text {
+      color: var(--requestly-color-text-subtle);
+      font-size: var(--requestly-font-size-xs);
+      font-style: normal;
+      font-weight: 400;
+      line-height: 130%; /* 14.3px */
+
+      .text-with-images {
+        display: flex;
+        gap: 8px;
+        a {
+          display: flex;
+          align-items: center;
+          gap: 4px;
+          flex-shrink: 0;
+        }
+        a > svg {
+          justify-content: center;
+        }
+      }
+
+      .red-text {
+        color: var(--requestly-color-error-text);
+      }
+
+      .blue-text {
+        color: var(--requestly-color-primary-text);
+      }
+    }
+
+    .content {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+      gap: 8px;
+      align-self: stretch;
+    }
+  }
+
+  .migrated-rules-item {
+    display: flex;
+    padding: 12px 16px;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 8px;
+    align-self: stretch;
+
+    border-radius: 4px;
+    background-color: var(--requestly-color-surface-1);
+
+    color: var(--requestly-color-text-default);
+    font-size: var(--requestly-font-size-sm);
+    font-style: normal;
+    font-weight: 400;
+    line-height: 150%; /* 19.5px */
+
+    ul {
+      padding: 0px;
+      margin: 0px;
+      margin-left: 16px;
+
+      li {
+        display: list-item;
+        list-style-type: disc;
+      }
+    }
+  }
+
+  .migrated-rule-name {
+    font-size: 16px;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    gap: 5px; /* Ensure proper spacing between icon and text */
+  }
+
+  .migrated-rule-description {
+    font-size: 14px;
+    color: #ccc;
+  }
+
+  /* Adjustments for Ant Design components if needed */
+  .ant-modal-body {
+    padding: 0; /* Remove padding if Ant Design adds any by default */
+  }
+
+  .ant-collapse-header {
+    color: white;
+  }
+
+  .ant-collapse-content {
+    background: #333;
+  }
```

Comment:
@nsrCodes Avoid targeting antd classes directly, this will affect all over the app.